src/amber/cli/templates/app/package.json.ecr

```diff
@@ -12,16 +12,20 @@
     "test": "echo \"No test specified\" && exit 1"
   },
   "devDependencies": {
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
-    "babel-preset-env": "^1.6.1",
+    "@babel/core": "^7.0.0-beta.38",
+    "@babel/preset-env": "^7.0.0-beta.38",
```

Comment:
Do we need this?

Reply:
So here is the dilemma.  If you look at https://github.com/babel/babel-preset-env, you will notice that the repo is archived.  If you go to the new place and follow instructions, it installs the beta.  It also seems from the babel-loader readme that it wants you to use the beta as well.  Do we want to go with the beta as the older way seems to be on its way out?

Reply:
I prefer we go with non-beta versions.  We will leave updating it as an exercise for the developer.